src/apps/chifra/internal/names/handle_autoname.go

```diff
@@ -1,22 +1,61 @@
 package namesPkg
 
-import "fmt"
+import (
+	"fmt"
+
+	"github.com/TrueBlocks/trueblocks-core/src/apps/chifra/pkg/base"
+	"github.com/TrueBlocks/trueblocks-core/src/apps/chifra/pkg/logger"
+	"github.com/TrueBlocks/trueblocks-core/src/apps/chifra/pkg/names"
+	"github.com/TrueBlocks/trueblocks-core/src/apps/chifra/pkg/rpcClient/ens"
+	"github.com/TrueBlocks/trueblocks-core/src/apps/chifra/pkg/types"
+)
 
 func (opts *NamesOptions) HandleAutoname() error {
```

Comment:
All handler routines need to send the result (on success) through the modelChan and any errors through the errChan so that the client gets the error message when running against the server.